api/libproject/devstart.rb

```diff
@@ -2543,62 +2519,10 @@ def create_project_resources(gcc)
   common.run_inline %W{gcloud app create --region us-central --project #{gcc.project}}
 end
 
-def setup_project_data(gcc, cdr_db_name)
-  root_password = random_password()
-  workbench_password = random_password()
-  readonly_password = random_password()
-
-  common = Common.new
-  # This changes database connection information; don't call this while the server is running!
-  common.status "Writing DB credentials file..."
-  write_db_creds_file(gcc.project, cdr_db_name, root_password, workbench_password, readonly_password)
-  common.status "Setting root password..."
-  run_with_redirects("gcloud sql users set-password root --host % --project #{gcc.project} " +
-                     "--instance #{INSTANCE_NAME} --password #{root_password}",
-                     root_password)
-  # Don't delete the credentials created here; they will be stored in GCS and reused during
-  # deployment, etc.
-  with_cloud_proxy_and_db(gcc) do
-    common.status "Setting up databases and users..."
-    create_or_update_workbench_db
-
-    common.status "Running schema migrations..."
-    migrate_database
-
-  end
-end
-
 def random_password()
   return rand(36**20).to_s(36)
 end
 
-# TODO: add a goal which updates passwords but nothing else
-# TODO: add a goal which updates CDR DBs but nothing else
-
-def setup_cloud_project(cmd_name, *args)
```

Comment:
Removed because I don't think these have been tested recently and would not trust them to work correctly.